package.json

```diff
@@ -0,0 +1,50 @@
+{
+  "name": "webpacktemplate",
+  "version": "1.0.0",
+  "description": "This  repo can be used for projects that use webpack bundling.",
+  "main:": "index.js",
+  "scripts": {
+    "start": "webpack --mode development --watch",
+    "build": "webpack --mode production",
+    "dev": "webpack serve --mode development --open --hot",
+    "test": "jest",
+    "watch": "jest --watch *.js",
+    "testdom": "jest --testEnvironment jsdom --testPattern='*.test.js"
```

Comment:
- You have done great so far :+1: :+1: Setting up webpack and the project structure. But there is an issue you need to fix before moving to the next milestone. :+1: 
- [ ] Looking at the video below, You can see that I didn't get the project to open in the browser. you can see that when I run the npm script `npm run dev`  it gives an error in the terminal. Then when I ran the start script `npm run start`, the server starts but didn't provide the URL or the port to view it in the browser. trying the most common ports didn't open it. Usually, when you run the start script, the project automatically opens in the browser, but if it didn't, it provides the URL for that, this is not the case here. I'' suggest you update your start script or include an instruction on which URL and POR to view your leaderboard project. :smile: :wave: :+1: 

- - VIDEO REFERENCE:

https://github.com/dvcorreia1984/leaderboard/assets/107632780/be5678df-543e-4c54-8a0e-92ba91ecad95